armada/handlers/armada.py

```diff
@@ -69,7 +69,7 @@ def find_release_chart(self, known_releases, name):
         '''
         Find a release given a list of known_releases and a release name
         '''
-        for chart_name, _, chart, values in known_releases:
+        for chart_name, _, chart, values, _ in known_releases:
```

Comment:
I just added in the '_' so that the method would function properly now that i changed the format of known_releases. it works exactly the same as before which is what I was going for but I can change the functionality if you want. what should it do differently @gardlt ?